packages/@aws-cdk/aws-cloudfront-origins/README.md

```diff
@@ -76,3 +76,20 @@ new cloudfront.Distribution(this, 'myDist', {
 ```
 
 See the documentation of `@aws-cdk/aws-cloudfront` for more information.
+
+## Failover Origins
+
+If you want to provide a failover Origin,
```

Comment:
*Nit:* Why the 40-character wrap? :)
```suggestion
If you want to provide a failover Origin
```

Reply:
This is my personal style: in m opinion, docs should end lines on sentence boundaries. This makes future changes to these docs have much better diff than if you just follow a set line length like 80 or 120.